CONTRIBUTING.md

```diff
@@ -5,38 +5,74 @@
 
-We hope that these guidelines are designed to make it as easy as possible to get involved. If you have any questions that aren't discussed below, please let us know through one of the many ways to [get in touch](#get-in-touch).
+We hope that these guidelines are designed to make it as easy as possible to get involved. If you have any questions that aren't discussed below, please let us know by [opening an issue](#understanding-issues).
 
 ## Table of contents
 
 Been here before? Already know what you're looking for in this guide? Jump to the following sections:
 
 *   [Joining the BIDS community](#joining-the-community)
-*   [Get in touch](#get-in-touch)
 *   [Contributing through GitHub](#contributing-through-github)
+*   [Understanding issues](#understanding-issues)
 *   [Writing in markdown](#writing-in-markdown)
 *   [Make a change with a pull request](#making-a-change-with-a-pull-request)
 *   [Example pull request](#example-pull-request)
 *   [Recognizing contributions](#recognizing-contributions)
 
 ## Joining the community
 
-BIDS - the [Brain Imaging Data Structure](http://bids.neuroimaging.io/) - is a growing community of neuroimaging enthusiasts, and we want to make our resources accessible to and engaging for as many researchers as possible.
-
-We therefore require that all contributions **adhere to our [Code of Conduct](CODE_OF_CONDUCT.md)**.
+BIDS - the [Brain Imaging Data Structure](https://bids.neuroimaging.io/) - is a growing community of neuroimaging enthusiasts, and we want to make our resources accessible to and engaging for as many researchers as possible.
 
 How do you know that you're a member of the BIDS community? You're here! You know that BIDS exists! You're officially a member of the community. It's THAT easy! Welcome!
 
-## Get in touch
+Most of our discussions take place here in [GitHub issues](#understanding-issues).
+We also have a [bids-discussion](https://groups.google.com/forum/#!forum/bids-discussion) Google Group, although this is largely now an archive of previous conversations.
+
+Moving forward, we encourage all members to contribute here on [GitHub](https://github.com/bids-standard/bids-specification) or on the [NeuroStars](https://neurostars.org/tags/bids) Discourse Forum, under the `bids` tag.
+
+To keep on top of new posts, please see this guide for setting your [topic notifications](https://meta.discourse.org/t/discourse-new-user-guide/96331#heading--topic-notifications).
+
+As a reminder, we expect that all contributions adhere to our [Code of Conduct](CODE_OF_CONDUCT.md).
 
 ## Contributing through GitHub
 
-[Git](https://git-scm.com/) is a really useful tool for version control. [GitHub](https://github.com/) sits on top of git and supports collaborative and distributed working.
+[git](https://git-scm.com/) is a really useful tool for version control. [GitHub](https://github.com/) sits on top of git and supports collaborative and distributed working.
```

Comment:
I don't know if there's an MLA convention or something, but I would capitalize at the beginning of a sentence, unless using backticks to indicate that we're referring specifically to the CLI `git`.